src/app/api/common/delegates/getDelegates.ts

```diff
@@ -95,149 +95,157 @@ async function getDelegates({
       )`
       : "";
 
+  const includeZeroVPDelegates = true;
+
+  let delegateUniverseCTE: string;
+
+  const tokenAddress = contracts.token.address;
+
+  if (includeZeroVPDelegates) {
```

Comment:
do we need this condition? seems like it's hardcoded to `true`. Also I don't really see a use case where we wouldn't include it

Reply:
It was only there as a backup.  to show the logic to undo this, if performance was terrible for one tenant.

Reply:
I'll back it out in the rebase.  This thread is going to get hidden when I do so.